### PR TITLE
Fix language issues

### LINKED ISSUE #80: Exception when add page with taxonomy behavior
```python
Traceback (innermost last):
  Module ZPublisher.WSGIPublisher, line 156, in transaction_pubevents
  Module ZPublisher.WSGIPublisher, line 338, in publish_module
  Module ZPublisher.WSGIPublisher, line 256, in publish
  Module ZPublisher.mapply, line 85, in mapply
  Module ZPublisher.WSGIPublisher, line 62, in call_object
  Module plone.z3cform.layout, line 63, in __call__
  Module plone.z3cform.layout, line 47, in update
  Module plone.dexterity.browser.add, line 141, in update
  Module plone.z3cform.fieldsets.extensible, line 65, in update
  Module plone.z3cform.patch, line 30, in GroupForm_update
  Module z3c.form.group, line 141, in update
  Module z3c.form.group, line 52, in update
  Module z3c.form.group, line 48, in updateWidgets
  Module z3c.form.field, line 277, in update
  Module collective.taxonomy.widget, line 39, in update
  Module z3c.form.browser.orderedselect, line 54, in update
  Module z3c.form.browser.orderedselect, line 54, in <listcomp>
  Module z3c.form.browser.orderedselect, line 45, in getItem
  Module zope.i18n, line 194, in translate
  Module collective.taxonomy.utility, line 296, in translate
KeyError: 'en-us'
```

### PR DESCRIPTION
**Pull request by petschki**
fixes #80

### CODE REVIEW

**Comment by petschki** [comment]
@rodfersou this is my work from yesterday ... feel free to test/change it

**Comment by rodfersou** [comment]
@petschki great! thanks

**Comment by rodfersou** [comment]
@petschki could you please just look my last commit if you agree?

File: src/collective/taxonomy/jsonimpl.py [hunk 1]
```diff
@@ -71,19 +67,8 @@ def get_data(self):
...
         language_tool = api.portal.get_tool('portal_languages')
-        supported_langs = language_tool.supported_langs
-        languages_mapping = {
-            get_lang_code(lang): mapping[get_lang_code(lang)].capitalize()
-            for lang in supported_langs
-        }
-        # add taxonomy's default language if it is not in supported langs
-        default_lang = self.taxonomy.default_language
-        languages_mapping[default_lang] = mapping[default_lang].capitalize()
+        languages_mapping = dict(language_tool.listAvailableLanguages())
```
Comment (petschki) [inline src/collective/taxonomy/jsonimpl.py:71]: erm ... you've remove the code which adds eventually existing taxonomies in no more existing portal languages ... I think we should still provide this *hidden* taxonomies in edit view

Comment (rodfersou) [reply src/collective/taxonomy/jsonimpl.py:71]: by reading the code could not understand what was going on.

will try to change variable names to make it clear and add a test

Comment (petschki) [reply src/collective/taxonomy/jsonimpl.py:71]: by simplifying that way, you get all the available languages on the edit view, not just the supported ones in the language settings ... this makes it hard to translate taxonomies in the correct supported language ... I'd recommend reverting the last commit ...

Comment (rodfersou) [reply src/collective/taxonomy/jsonimpl.py]: done

**Comment by rodfersou** [comment]
@petschki I would like to merge this when get green if you agree :)

**Comment by rodfersou** [comment]
green!

**Pull request merged**